test/Generator.Tests.Generated/EnumOutputComplexCommand.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Generator.Tests.Generated
+{
+    using System.ComponentModel.DataAnnotations;
+    using System.Runtime.Serialization;
+    using System.Text.Json.Serialization;
+
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum EnumOutputComplexCommand
```

Comment:
Come to think of it this should probably be renamed to AssetComplexCommandResponseEnum. @jarz what do you think?

Reply:
I _think_ it should be `AssetComplexCommandResponse` -- `POISubStatus` is another `enum` that's generated.

Reply:
Can an Enum be nested in an object response?